src/d_iwad.c

```diff
@@ -28,20 +28,20 @@
 #include "m_misc.h"
 
 static const iwad_t iwads[] = {
-    {"doom2.wad",     doom2,      commercial,   "Doom II"                        },
-    {"plutonia.wad",  pack_plut,  commercial,   "Final Doom: Plutonia Experiment"},
-    {"tnt.wad",       pack_tnt,   commercial,   "Final Doom: TNT: Evilution"     },
+    {"doom2.wad",     doom2,      commercial,   "DOOM II: Hell on Earth"         },
+    {"plutonia.wad",  pack_plut,  commercial,   "Final DOOM: Plutonia Experiment"},
+    {"tnt.wad",       pack_tnt,   commercial,   "Final DOOM: TNT - Evilution"    },
```

Comment:
But these strings were taken from prboom+, right? I remember changing one of the colons into a dash years ago.

Reply:
These lines are taken from the original DOS Doom source (I changed some things, like "DOOM 2"->"Final DOOM").

Reply:
I mixed it up with this commit then: 

https://github.com/coelckers/prboom-plus/commit/7e442f25884a8f6a8105a912ec0e031977439672

Reply:
Original: https://bitbucket.org/gamesrc-ver-recreation/doom/src/bd6beabfd561613b671fe5b8842e7d373731d2c5/d_main.c#lines-1542